pkg/kubelet/config/file_linux_test.go

```diff
@@ -250,7 +223,18 @@ func (tc *testCase) writeToFile(dir, name string, t *testing.T) string {
 	return fileName
 }
 
-func watchFileAdded(watchDir bool, t *testing.T) {
+func (tc *testCase) writeSymblicLink(dir, linkedDir, name string, t *testing.T) string {
```

Comment:
Also please use clearer argument names to indicate the direction of the link, e.g. `sourcedir`, `destdir`.